aws/lambda/blueprint.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: AWS Lambda and Credentials
+  description: |
+    This blueprint will configure one AWS Lamba application and one AWS credential set.
+  author: XebiaScrubs
+  version: 1.0
+parameters:
+- name: AwsInfrastructureDirectory
+  type: Input
+  description: "Infrastructure directory (default: AmazonAWS)"
+  default: "AmazonAWS"
+- name: AwsInfrastructureName
+  type: Input
+  description: "Infrastructure credential pair name (default: AmzAccount)"
+  default: AmzAccount
+- name: AwsEnvironmentDirectory
+  type: Input
+  description: 'Environment directory (default: aws-staging)'
+  default: aws-staging
+- name: AwsEnvironmentName
+  type: Input
+  description: 'Environment name (default: "AWSHosts")'
+  default: AWSHosts
+- name: AwsAccessKeyID
+  type: Input
+  description: AWS Access Key ID (example:AMZ1dF4Nz7)'
+- name: AwsAccessSecret
+  type: Input
+  description: 'AWS Region (example: wt9JK8VU5ue45)'
+- name: AwsRegion
```

Comment:
use a Select type and get this list using inbuilt function, see an existing aws blueprint for sample https://github.com/xebialabs/blueprints/blob/development/aws/monolith/blueprint.yaml